hms-app/src/pages/HomeQandAContent.tsx

```diff
@@ -0,0 +1,42 @@
+export const qAndAList = [
```

Comment:
not a page right? so maybe should be moved to the common folder or something like this